src/lib/radio/OVERVIEW.md

```diff
@@ -0,0 +1,29 @@
+`<md-checkbox>` provides the same functionality as a native `<input type="radio">` enhanced with 
```

Comment:
Shouldn't this be `<md-radio-button>`? There's another instance a little bit below.